### ISSUE TITLE
It downloads only html asking for CAPCHA

### ISSUE THREAD

**Issue by Neo2003** [TOLoneWolf/curseDownloader#5]
Part of downloaded files:
```
<head>
<title>Attention Required! | Cloudflare</title>
<meta name="captcha-bypass" id="captcha-bypass" />
```
I am not sure this can anymore be used to download modpacks which is pain for Linux users since CurseClient does not work anymore in Lutris.

**Comment by TOLoneWolf** [comment]
yeah this hasn't been updated in a while. it could be redone to use the curse api that is now somewhat public.
There are better options atm.
CMPDL downloads packs like this does.
https://github.com/Franckyi/CMPDL

and a full client that does download and runnning /manage clients. (Win/OSX/Linux)
https://github.com/gorilla-devs/GDLauncher

And if your building packs, this tool can be used to pack them for twitch use.
https://github.com/Gaz492/twitch-export-builder




GDlauncher runs on
